shared/src/test/java/TestStatsProcessor.java

```diff
@@ -57,4 +57,18 @@ public void getLeaderboard() {
         assertThat(leaderboard[1].getPlayerName(), equalTo("CHS"));
         assertThat(leaderboard[1].getHighestScore(), equalTo(18));
     }
+
+    @Test
+    public void getEmptyAllStats() {
+        Database db = Database.create("concrete", "dummyFilename");
```

Comment:
This will create a test file at `shared/dummyFilename`. Can you change it to be `src/test/resources/tempTestDb.csv` like the other file tests?